docs/developer-docs/integrations/sns/integrate-sns/index-integration.md

```diff
@@ -1 +1,51 @@
-Title h1: Considerations to Integrate with Index Canister
\ No newline at end of file
+# Index Canister
+
+The index canister keeps track of each transaction stored by the 
+[ledger canister](ledger-integration.md).
+
+At each heartbeat, the index canister will query the transactions from
+the ledger canister, then build the index
+of known transaction per account.
+
+## Initialisation
+
+The index canister initialisation requires the principal
+of the ledger canister to index:
+
+```
+type InitArgs = record {
+ledger_id : principal;
+};
+```
+
+Example with `dfx`:
```

Comment:
Should we say here that this is an example how the index canister can be initialised if it is deployed in isolation? 
I would then also say that "you can also deploy a full SNS, which will be deployed with an SNS index canister" and link to the respective page where this is described (local-testing)

Reply:
Made explicit that this is a stand-alone initialization command.